_config.yml

```diff
@@ -0,0 +1,17 @@
+title: Isthmus Carshare
+email: isthmuscarshare@gmail.com
+description: >- # this means to ignore newlines until "baseurl:"
+  Write an awesome description for your new site here. You can edit this
```

Comment:
Maybe we should put our description from the front page here?

An affordable, member-owned, and community-oriented alternative to private car ownership in Madison that will enhance mobility and transportation equity in our city, reduce congestion, and promote environmental stewardship

Reply:
updated